Tests/scripts/validate_files_structure.py

```diff
@@ -145,6 +156,7 @@ def get_modified_files(files_string):
         file_status = file_data[0]
 
         if file_status.lower().startswith("r"):
+            file_status = "r"   # usually it will be R099 - but we can ignore the number
```

Comment:
the number after R, represents the percentage of diff between the files, so it can really be any R from 0 to 100

Reply:
I know, that is why I ignore that number

Reply:
I ment the comment might be more indicative if it explains the number after R instead of giving 099 as example which might confuse people.